docker-compose.yml

```diff
@@ -20,7 +20,18 @@ services:
     environment:
       MYSQL_ROOT_PASSWORD: password
     restart: unless-stopped
- 
+
+  mongodb:
+    image: mongo:latest
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: password
+    ports:
+      - 27017:27017
+    volumes:
+      - /data/db
```

Comment:
can this be run from a different folder? with the mysql one we store the data in `./db` - it makes the docker image portable

Reply:
Done!